deployments/engine/docker-compose/config/master_with_tls.toml

```diff
@@ -0,0 +1,27 @@
+etcd-endpoints = ["etcd-standalone:2379"]
+
+[framework-meta]
+endpoints = ["mysql-standalone:3306"]
+schema = "test_framework"
+user = "root"
+password = "123456"
+
+[framework-meta.security]
+ca-path = "/ca.pem"
```

Comment:
Does MySQL support verify client certificates?

Reply:
Yes, I have added the test for it in https://github.com/pingcap/tiflow/blob/6587d4ebb46baeeecf14158a14385f6f7870ef74/engine/test/integration_tests/dm_tls/run.sh#L27-L28 , in dm_tls I created MySQL user by `create user 'dm_user'@'%' require X509;`

And I just found forget to change TiDB user name to `dm_user`. Will fix it in this PR later

Reply:
Can we let meta security also use it?

Reply:
- "only providing CA" is tested by engine connecting to meta
- "only providing client certificates" is tested by DM business

if we change the test there's no chance to cover https://github.com/pingcap/tiflow/issues/5335. And since meta store is controlled by us, I think we will not use CREATE USER REQUIRE X509 in future, no need to test it.